pkg/util/yolo.go

```diff
@@ -1,12 +0,0 @@
-// SPDX-License-Identifier: AGPL-3.0-only
-// Provenance-includes-location: https://github.com/cortexproject/cortex/blob/master/pkg/util/yolo.go
-// Provenance-includes-license: Apache-2.0
-// Provenance-includes-copyright: The Cortex Authors.
-
-package util
-
-import "unsafe"
-
-func YoloBuf(s string) []byte {
```

Comment:
Decided against exporting this, as it still passes on certain usage requirements to its callers and yet the linter doesn't know to look for call sites. Exporting something that directly wraps a linter check felt against the spirit of the lint rule.